### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
### BugFixes
- Fixed: 1091073 | Fixed the issue of Close tab button in `Configure the Feed Config` page
- Fixed: 1091073 | Added colon in `Total connector 2` label in `Install Feed Integration` page
- Fixed: 1092490 | The ingestion schedule has been updated with incorrect configuration settings.
- Fixed: 1092485 | The save button for the ingestion schedule gets stuck on "saving" indefinitely
- Fixed: 1092476 | Fortinet Fortiguard Connector is not marked for default configuration.
- Fixed: 1092388 | After closing all the connector tabs on the configure feed page, user is still able to proceed without connector config
- Fixed: 1092014 | Single schedule is getting created for multiple configuration may it be self or agent. 
- Fixed: 1091583 | User is not able to change Ingestion parameters which are already been chaanged once, hence the save button is disabled.

### Test Case:
- Tested the `Close Tab` button matches to all the theme
- Tested colon in `Total connector 2` label in `Install Feed Integration` page
- Tested Fortinet Fortiguard Connector is not marked for default configuration.
- Tested all schedule modification belongs to its associated connector config
- Every configuration separate schedule is crated 
- After closing all the tabs then added toaster `At least one integration configuration should be necessary.`
- User can change the ingestion parameters  multiple times

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -142,18 +142,40 @@
...
+        $scope.$apply(); // Apply changes
+        loadActiveTab(0);
+        const tabHeadings = document.querySelectorAll('.uib-tab-heading');
+        if(tabHeadings.length > 0){
+          tabHeadings[0].focus();
+        }
+      }, 2000);
+      $scope.params = {
+        activeTab: 0
+      };
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: add this also in timeout to keep consistency of active tab 0

File: widget/view.controller.js [hunk 2]
```diff
@@ -437,7 +488,7 @@
               const metaData = {
                 "name": $scope.saveSchedules[index].name,
                 "description": "Metadata for " + $scope.saveSchedules[index].description,
-                "modified_by": "3451141c-bac6-467c-8d72-85e0fab569ce",
+                "modified_by": "3451141c-bac6-467c-8d72-85e0fab569ce", //CSAdmin user uuid
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js:491]: this will be hardcoded ?

Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:491]: CsAdmin is default user and it's uuid is same in all instance

File: widget/view.controller.js (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js]: done

**Review by cs-pranali-chavan — APPROVED** [review]

File: widget/view.controller.js [hunk 3]
```diff
@@ -386,26 +429,34 @@
...
-      widgetDataIngestionService.saveDataIngestionParams($scope, timParamsForm, index)
+      widgetDataIngestionService.saveDataIngestionParams($scope, timParamsForm, index);
+      timParamsForm.$setPristine();
     }
 
 
     function _checkConnectorHealth() {
+      if ($scope.installedConnectors.length === 0) {
+        toaster.error({
+          body: 'At least one integration configuration should be necessary.'
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:456]: please review all toaster messages with Abhishek

**Pull request merged**